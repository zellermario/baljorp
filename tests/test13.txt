create ST m 0

create Sarkkutato s m

useskill s m
pass s