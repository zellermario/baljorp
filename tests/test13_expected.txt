Fields:
SJ m (0) - object: x
	neighbours: []

Buildings:

Actors:
> Sarkkutato s (heat: 4) on m
  Inventory: []